server/metrics/metrics.go

```diff
@@ -128,6 +131,65 @@ var (
 		CacheTypeLabel,
 	})
 
+	/// ## Remote execution metrics
+
+	RemoteExecutionCount = promauto.NewCounterVec(prometheus.CounterOpts{
+		Namespace: ns,
```

Comment:
nit: for stuff like this that is a const, it's probably clearer to just call it buildbuddyNamespace or bbNamespace

Reply:
Done